### PR TITLE
Mo homework

### PR DESCRIPTION
**Pull request by Artificial-Friend**

### CODE REVIEW

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        Bet bet;
+        while (true) {
+            if (command.equalsIgnoreCase("q")) {
+                return;
+            }
+            try {
+                betData = command.split(" ");
+                value = Integer.parseInt(betData[0]);
+                risk = Double.parseDouble(betData[1]);
+                bet = new Bet(value, risk);
```
Comment (Vedroid) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]:
```suggestion
        String command = scanner.nextLine();
        while (true) {
            if (command.equalsIgnoreCase("q")) {
                return;
            }
            try {
                String[] betData = command.split(" ");
                int value = Integer.parseInt(betData[0]);
                double risk = Double.parseDouble(betData[1]);
                Bet bet = new Bet(value, risk);
```

File: src/main/java/core/basesyntax/dao/UserDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.List;
+
+public class UserDaoImpl implements UserDao {
+    @Override
+    public List<Bet> getBets() {
```
Comment (Vedroid) [inline src/main/java/core/basesyntax/dao/UserDaoImpl.java]: If you have UserDao then you should work with User

File: src/main/java/core/basesyntax/dao/BetDaoImpl.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import core.basesyntax.model.User;
+
+public class BetDaoImpl implements BetDao {
+    @Override
+    public void addBet(Bet bet) {
+        Storage.bets.add(bet);
+    }
+
+    @Override
+    public void addUser(User user) {
```
Comment (Vedroid) [inline src/main/java/core/basesyntax/dao/BetDaoImpl.java]: If you have BetDao then you should work with Bet

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    Scanner scanner = new Scanner(System.in);
```
Comment (Vedroid) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: You have not closed the resource

**Review by Vedroid — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java (hunk 1 above)
Comment (Artificial-Friend) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java]: Can you explain why this is better?

Comment (Vedroid) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java]: You must declare variables before using.
You don't use them outside the try block

**Review by Vedroid — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -28,4 +28,3 @@
 Не нужно создавать дополнительных аннотаций! У вас должно быть всего две аннотации - `@Dao` и `@Inject`.
 
 В конце второй части также создать Pull request, и закинуть на проверку.
-
```
Comment (dmytro-hryhoruk) [inline README.md]: ?

File: src/main/java/core/basesyntax/Main.java [hunk 6]
```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import core.basesyntax.controller.ConsoleHandler;
+
+public class Main {
+    public static void main(String[] args) {
+        ConsoleHandler handler = new ConsoleHandler();
+        System.out.println("Enter 'value' and 'risk' for your bid");
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/Main.java]: move this to handle() method

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+    BetDao dao = new BetDaoImpl();
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: What's is missing? and what about UserDao?

**Review by dmytro-hryhoruk — CHANGES_REQUESTED** [review]

**Comment by Artificial-Friend** [comment]
I will revert changes in README.md later. For now - I want to done the task as fast as possible.

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**